labs/preparation/data_encoding_Services.py

```diff
@@ -129,6 +144,8 @@ def encode_services():
     df = handle_months(df)
     df.drop(columns=["Month", "Type_of_Loan"], inplace=True)
 
+    df = remove_negatives(df, exclude_cols=["Customer_ID", "Month_cos", "Month_sin"])
```

Comment:
Should also exclude:

- Delay_from_due_date (as I think people have payed before due date, but it could also be a data error, if 0 is "before")
- ChangedCreditLimit (you can have less credit from this month on)